flutter/shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -354,6 +375,25 @@ void TizenViewElementary::PrepareInputMethod() {
       [this]() { view_delegate_->OnComposeEnd(); });
   input_method_context_->SetOnCommit(
       [this](std::string str) { view_delegate_->OnCommit(str); });
+#ifndef WEARABLE_PROFILE
+  input_method_context_->SetOnPopupAutofillContext([this]() {
+    if (!TizenAutofill::GetInstance().GetResponseItems().empty()) {
+      for (auto& item : TizenAutofill::GetInstance().GetResponseItems()) {
+        elm_ctxpopup_item_append(
+            ctxpopup_, item->label_.c_str(), nullptr,
+            [](void* data, Evas_Object* obj, void* event_info) {
+              AutofillItem* item = static_cast<AutofillItem*>(data);
+              TizenAutofill::GetInstance().OnCommit(item->value_);
+              evas_object_hide(obj);
+            },
+            item.get());
+      }
+    }
+    // TODO(Swanseo0): Change ctxpopup's position to focused input field.
+    evas_object_move(ctxpopup_, 0, 0);
```

Comment:
Isn't the popup supposed to appear just above the keyboard?

Reply:
We can set it to appear just above the input panel. Should I temporarily set it to appear just above the input panel?